.github/workflows/promote.yml

```diff
@@ -51,7 +51,11 @@ jobs:
           echo "Getting the manifest of the image tagged main-$SHA_SHORT"
           MANIFEST="$(curl -sS -H "Authorization: Basic $TOKEN" -H "Accept: $CONTENT_TYPE" "https://$ECR_REPO_DOMAIN/v2/$ECR_REPO/manifests/main-$SHA_SHORT")"
 
+          if [ "${{ inputs.environment }}" == "prod_test" ]; then
+          SHA_TAG="ab2d-prod-test-$SHA_SHORT"
+          else
           SHA_TAG="ab2d-$DEPLOYMENT_ENV-$SHA_SHORT"
+          fi
           echo "Adding the $SHA_TAG tag to main-$SHA_SHORT image"
           curl -sS -X PUT -H "Authorization: Basic $TOKEN" -H "Content-Type: $CONTENT_TYPE" -d "$MANIFEST" "https://$ECR_REPO_DOMAIN/v2/$ECR_REPO/manifests/$SHA_TAG"
 
```

Comment:
The LATEST_TAG below will need to be updated as well. Maybe better to create a TAG_PREFIX variable that is the DEPLOYMENT_ENV except when the environment is prod_test.